rmf_site_editor/src/interaction/gizmo.rs

```diff
@@ -282,10 +282,13 @@ pub fn update_gizmo_click_start(
         }
     }
 
-    let clicked = mouse_button_input.just_pressed(MouseButton::Left)
+    // Ignore if button was pressed and released in the same frame, to avoid being stuck in
+    // dragging behavior in cases when the frame rate is low.
+    let clicking = (mouse_button_input.just_pressed(MouseButton::Left)
+        && !mouse_button_input.just_released(MouseButton::Left))
         || touch_input.iter_just_pressed().next().is_some();
```

Comment:
Is it possible to have a similar problem with the touch input?

We aren't trying very hard to support touch inputs so I'm fine with ignoring this for now, it's just a thought that occurred to me.

Reply:
I am honestly not sure since I don't have a touch device handy. The system in charge of disabling dragging does actually not even listen to touch inputs so I'm not sure how that would have worked

Reply:
That seems problematic. Maybe we should just remove touch input here to be safe.